x-pack/plugins/security_solution/public/detection_engine/rule_gaps/translations.ts

```diff
@@ -125,3 +125,20 @@ export const BACKFILL_TABLE_SUBTITLE = i18n.translate(
     defaultMessage: 'View and manage backfill runs',
   }
 );
+
+export const BACKFILL_SCHEDULE_SUCCESS = (numRules: number) =>
+  i18n.translate(
+    'xpack.securitySolution.containers.detectionEngine.backfillSchedule.scheduleRuleRunSuccessTitle',
+    {
+      values: { numRules },
+      defaultMessage:
+        'Successfully scheduled backfill for {numRules} {numRules, plural, =1 {rule} other {rules}}',
```

Comment:
This syntax looks slightly different than the translation above with `maxDaysLookback`. Are they still equivalent?

Reply:
Yeah they are equivalent at the end, but I will update to use the same style. The special # character above will be replaced by the count variable when we resolve the message.